io_scene_nif/modules/nif_import/collision/havok.py

```diff
@@ -154,8 +154,8 @@ def import_bhk_ridgid_body(self, bhk_shape):
     def _import_bhk_rigid_body(self, bhkshape, collision_objs):
         # set physics flags and mass
         for b_col_obj in collision_objs:
-            scn = bpy.context.scene
-            scn.objects.active = b_col_obj
+            # todo [collision] make low level, remove operator!
```

Comment:
What is required here?

Reply:
Avoid accessing the context object + using the operator, in the lines below the comment